src/physics/manufactured_solution.cpp

```diff
@@ -213,16 +217,18 @@ ::value(const dealii::Point<dim,real> &point, const unsigned int istate) const
 {
     real value = 0.0;
     if (dim == 2) {
-        const real rho = primitive_value(point,0);
-        const real u   = primitive_value(point,1);
-        const real v   = primitive_value(point,2);
-        const real p   = primitive_value(point,3);
+        const real rho = primitive_value(point,0); // density
+        const real u   = primitive_value(point,1); // x-velocity
+        const real v   = primitive_value(point,2); // y-velocity
+        const real p   = primitive_value(point,3); // pressure
+        const real twv = primitive_value(point,4); // turbulent working variable (twv)
```

Comment:
Several variables have been abbreviated. If it doesn't make the file too verbose, writing the complete name might be a good idea.

Reply:
fixed